PowerPointLabs/PowerPointLabs/PasteLab/PasteLabMain.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Windows;
+
+using PowerPointLabs.ActionFramework.Common.Log;
+using PowerPointLabs.Utils;
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.PasteLab
+{
+    public class PasteLabMain
+    {
+        public static void PasteToFillSlide(Models.PowerPointSlide slide, bool clipboardIsEmpty, float width, float height)
+        {
+            if (clipboardIsEmpty)
+            {
+                Logger.Log("PasteToFillSlide encountered empty clipboard");
+                return;
+            }
+
+            PowerPoint.ShapeRange pastedObject = slide.Shapes.Paste();
+
+            Logger.Log(string.Format("PasteToFillSlide: {0} objects pasted", pastedObject.Count));
+
+            for (int i = 1; i <= pastedObject.Count; i++)
+            {
+                var shape = new PPShape(pastedObject[i]);
+                shape.AbsoluteHeight = height;
+                shape.AbsoluteWidth = width;
+                shape.VisualTop = 0;
+                shape.VisualLeft = 0;
+            }
+        }
+        
+        public static bool IsClipboardEmpty()
```

Comment:
@kai33 How to go about ensuring that this piece of code can be reused? There's multiple places where this check is done (for instance, to determine if the button is enabled)